lib/wasmopt.ts

```diff
@@ -1,10 +1,7 @@
 // Copyright 2018-2024 the Deno authors. MIT license.
 
-import { Buffer } from "@std/io/buffer";
-import { Untar } from "@std/archive/untar";
-import { copy } from "@std/io/copy";
+import { UntarStream } from "https://raw.githubusercontent.com/denoland/std/0352a3c441aefe39d1233b4be92c49e404963457/tar/untar_stream.ts";
```

Comment:
Can this be `@std/tar/untar-stream` now?

Reply:
Fixed now.